### PR TITLE
Response text field can be hidden for certain channels and other requested changes

### PR DESCRIPTION
**Pull request by Sander-Kornev**
in ALChatViewController with ["AL_ADMIN_BROADCAST": "true"] metadata. Removed UIFontMetrics from dynamic fonts.

### CODE REVIEW

File: sample-with-framework/Applozic/Applozic.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -2094,7 +2094,7 @@
 				GCC_PREFIX_HEADER = Applozic/Applozic_Prefix.pch;
 				INFOPLIST_FILE = Applozic/Info.plist;
 				INSTALL_PATH = "$(LOCAL_LIBRARY_DIR)/Frameworks";
-				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
```
Comment (Sunilkumarr) [inline sample-with-framework/Applozic/Applozic.xcodeproj/project.pbxproj]: Revert the IPHONEOS_DEPLOYMENT_TARGET to 8.0 there are many APIS will be deprecated if it's 9.0

File: sample-with-framework/Applozic/Applozic.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -2128,7 +2128,7 @@
...
+				IPHONEOS_DEPLOYMENT_TARGET = 9.0;
```
Comment (Sunilkumarr) [inline sample-with-framework/Applozic/Applozic.xcodeproj/project.pbxproj]: same here

File: sample-with-framework/applozicdemo.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -586,7 +586,7 @@
 					"$(PROJECT_DIR)",
 				);
 				INFOPLIST_FILE = applozicdemo/Info.plist;
-				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 9.0;
```
Comment (Sunilkumarr) [inline sample-with-framework/applozicdemo.xcodeproj/project.pbxproj]: @Sander-Kornev here as well keep 8.0

**Comment by Sander-Kornev** [comment]
@Sunilkumarr Why don't you consider upgrading Deployment Target to iOS 9 like it was done for your other library https://github.com/AppLozic/ApplozicSwift ?

`there are many APIS will be deprecated if it's 9.0`
So it would be a good opportunity to keep up with latest development and not stick to deprecated API, isn't it?

**Comment by Sunilkumarr** [comment]
Why don't you consider upgrading Deployment Target to iOS 9 like it was done for your other library https://github.com/AppLozic/ApplozicSwift ?

there are many APIS will be deprecated if it's 9.0
So it would be a good opportunity to keep up with latest development and not stick to deprecated API, isn't it?
--> We already started working on migrating to target 9.0 will be available in next updates.  I don't think the change in this commit is not required

**Comment by Sander-Kornev** [comment]
@Sunilkumarr Take a look. https://github.com/AppLozic/Applozic-iOS-SDK/pull/488/commits/c8e66db85f0cfe8c2c2eb6a323ca5bf8cfc83c9e

**Comment by Sunilkumarr** [comment]
Thanks for PR , 
Closing this PR we have pushed this changes in our internal  repo will be merged in our release

**Comment by Sander-Kornev** [comment]
@Sunilkumarr  thanks

**Pull request closed without merging**